src/libs/antares/timeelapsed/aggregator.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <yuni/yuni.h>
+#include <yuni/string.h>
+
+#include "info.h"
+#include "i_writer.h"
+
+namespace TimeElapsed
+{
+class Aggregator final
```

Comment:
For more expressiveness, maybe we should call that class **TimeMeasureFileContentHandler** (ok, quite long), or **FileContentHandler** (inside name space **TimeElapsed**)